### PR TITLE
Show Link to Goerli Etherscan Post-Deposit

### PR DESCRIPTION
**Pull request by rauljordan**
This PR adds a suggestion to check out the goerli tx for a deposit on etherscan, which has deep linking to their eth2 testnet on their site

### CODE REVIEW

File: src/app/pages/participate/participate.component.html [hunk 1]
```diff
@@ -155,18 +155,14 @@ <h1>
...
       <br/>Once you get assigned,
       you'll be able to see how your validator performs its responsibility of creating or voting on blocks as well as earning any rewards throughout its lifecycle.
       Your beacon chain client must be synced for the validator assignment to be seen by your validator client.
       If you leave your validator offline for a while, it will begin to accrue penalties from the network for being idle. Wanna learn more about Prysm and the
-      state of the testnet? Join our discord channel <a href="https://discord.gg/YMVYzv6" target="_blank">here</a> to chat! 
-    </div>
-
-    <div class="activation-status" *ngIf="validatorStatus">
-      <h4>Activation Status: {{validatorStatus.state}}</h4>
-      <mat-progress-bar mode="buffer" [value]="validatorStatus.percent" [bufferValue]="validatorStatus.percent"></mat-progress-bar>
```
Comment (prestonvanloon) [inline src/app/pages/participate/participate.component.html:169]: Whats up with this?

Comment (0xKiwi) [reply src/app/pages/participate/participate.component.html:169]: Activation takes several hours, not much purpose in us displaying it.

File: src/app/pages/participate/participate.component.html [hunk 2]
```diff
@@ -155,18 +155,14 @@ <h1>
   <mat-step>
     <ng-template matStepLabel>Wait for your validator assignment</ng-template>
     <div>
-      Now, once your validator is up and running - you just have to wait for its activation! This process takes about 2 hours. You do not have to leave this window open after you submit your transaction. 
-      You can follow the current beacon chain in Bitfly's block explorer: <a href="https://beaconcha.in" target="_blank">https://beaconcha.in</a> or on <a href="https://beacon.etherscan.io" target="_blank">https://beacon.etherscan.io</a>. Additionally, we recommend registering your node on <a href="https://eth2stats.io" target="_blank">eth2stats</a>.
```
Comment (0xKiwi) [inline src/app/pages/participate/participate.component.html:159]: Why remove this part? Good to send users to the explorer sites no?

**Comment by rauljordan** [comment]
PTAL @0xKiwi

**Review by 0xKiwi — APPROVED** [review]

**Pull request merged**